Cogs.Publishers/OwlPublisher.cs

```diff
@@ -50,14 +53,24 @@ public void Publish(CogsModel model)
             var projName = model.Settings.Slug;
             StringBuilder res = new StringBuilder();
             res.AppendLine(@"<?xml version=""1.0""?>");
-            res.AppendLine(@"<rdf:RDF xmlns=""" + TargetNamespacePrefix + projName + @"""");
-            res.AppendLine(@"   xml:base=""" + TargetNamespacePrefix + projName + @"""");
+            res.AppendLine(@"<rdf:RDF xmlns=""" + TargetNamespace +@"""");
+            res.AppendLine(@"   xml:base=""" + TargetNamespacePrefix + @"""");
             res.AppendLine(@"   xmlns:rdf=""http://www.w3.org/1999/02/22-rdf-syntax-ns#""");
             res.AppendLine(@"   xmlns:owl=""http://www.w3.org/2002/07/owl#""");
             res.AppendLine(@"   xmlns:xml=""http://www.w3.org/XML/1998/namespace""");
             res.AppendLine(@"   xmlns:xsd=""http://www.w3.org/2001/XMLSchema#""");
             res.AppendLine(@"   xmlns:rdfs=""http://www.w3.org/2000/01/rdf-schema#"">");
-            res.AppendLine(@"   <owl:Ontology rdf:about=""" + TargetNamespacePrefix + projName + @"""/>");
+            if(VersionInfo != null || Description != null)
+            {

```

Comment:
put the null checks around each property, so we write the ones which are present out even when one might be null